python/private/proto/BUILD.bazel

```diff
@@ -37,6 +37,23 @@ bzl_library(
 
 proto_lang_toolchain(
     name = "python_toolchain",
+    # proto_common will generate extraneous pb2 files for the built-in proto files unless we blacklist them here
+    # these proto files already have implementation in the runtime which we are implicitly adding as a dependency
+    # to all py_proto_libraries
+    blacklisted_protos = [
```

Comment:
Can you add a comment to hint at how to determine what should or shouldn't be in this list? I'm guessing something somewhere in com_google_protobufs has them as a dependency somewhere already?